tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/test_rsc_compile.py

```diff
@@ -110,8 +110,7 @@ def test_no_dependencies_between_scala_and_java_targets(self):
       print(dependee_graph)
       self.assertEqual(dedent("""
                      zinc[zinc-java](java/classpath:java_lib) <- {}
-                     rsc(scala/classpath:scala_lib) <- {}
-                     zinc[rsc-and-zinc](scala/classpath:scala_lib) <- {}""").strip(),
+                     zinc[zinc-only](scala/classpath:scala_lib) <- {}""").strip(),
```

Comment:
This probably represents a bug, but can fix in #8109. Probably need to set the workflow here.